modules/subscription/modules/subscription/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "@modules/subscription",
+    "version": "1.0.1",
+    "description": "Subscription Module",
+    "private": true,
+    "main": "index.js",
+    "author": "Crowdbotics",
+    "license": "ISC",
+    "dependencies": {
+    },
+    "x-dependencies": {
+    },
+    "scripts": {
+        "postinstall": "cd ../../../../ && yarn run add payments && yarn run add react-native-login"
```

Comment:
This won't work in our module installer process, we do have a peer dependent modules feature in the backlog for this.